opencti-platform/opencti-graphql/src/domain/xtm-hub.ts

```diff
@@ -65,3 +75,50 @@ export const checkXTMHubConnectivity = async (context: AuthContext, user: AuthUs
 
   return { status: newRegistrationStatus };
 };
+
+export const autoRegisterOpenCTI = async (context: AuthContext, input: AutoRegisterInput): Promise<{ success: boolean; } > => {
+  const xtmHubUrl = nconf.get('xtm:xtmhub_url');
+  const settings = await getEntityFromCache<BasicStoreSettings>(context, SYSTEM_USER, ENTITY_TYPE_SETTINGS);
+
+  const response = await fetch(`${xtmHubUrl}/graphql-api`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      'XTM-Hub-Platform-Token': input.platform_token ?? '',
```

Comment:
Do we really want to have a header with emtpy string for hub token ? I think it would be better to throw an exception if `input.platform_token` is empty. It's useless to call xtm hub in that case i don't think it will work.